json-examples/external-ot-shelf.json

```diff
@@ -0,0 +1,1195 @@
+                            "otsi-ref": [
+                              2
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "link-id": "Add-Drop-Bundled-Link-Forward",
+            "source": {
+              "source-node": "WDM-TE-Node-1",
+              "source-tp": "23"
+            },
+            "ietf-te-topology:te": {
+              "bundled-links": {
+                "bundled-link": [
+                  {
+                    "sequence": 1,
+                    "src-tp-ref": "2"
+                  },
+                  {
+                    "sequence": 2,
+                    "src-tp-ref": "3"
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "link-id": "Add-Drop-Bundled-Link-Reverse",
+            "destination": {
+              "dest-node": "WDM-TE-Node-1",
+              "dest-tp": "23"
+            },
+            "ietf-te-topology:te": {
+              "bundled-links": {
+                "bundled-link": [
+                  {
+                    "sequence": 1,
+                    "des-tp-ref": "2"
+                  },
+                  {
+                    "sequence": 2,
+                    "des-tp-ref": "3"
+                  }
+                ]
+              }
+            }
+          }
+        ]
+      },
+      {
+        "network-id": "WDM-Network-2",
+        "network-types": {
+          "ietf-te-topology:te-topology": {
+            "ietf-optical-impairment-topology:optical-impairment-topology": {}
+          }
+        },
+        "ietf-te-topology:te-topology-identifier": {
+          "topology-id": "WDM-Network-2"
+        },
+        "ietf-te-topology:te": {},
+        "ietf-optical-impairment-topology:otsi-group": [
+          {
+            "otsi-group-id": "Red OTSiG (Forward)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              }
+            ]
+          },
+          {
+            "otsi-group-id": "Red OTSiG (Reverse)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              }
+            ]
+          },
+          {
+            "otsi-group-id": "Green OTSiG (Forward)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              },
+              {
+                "otsi-carrier-id": 2
+              }
+            ]
+          },
+          {
+            "otsi-group-id": "Green OTSiG (Reverse)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              },
+              {
+                "otsi-carrier-id": 2
+              }
+            ]
+          }
+        ],
+        "node": [
+          {
+            "node-1": "WDM-TE-Node-2",
```

Comment:
"node-1" --> node-id

Reply:
Fixed in https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/127/commits/9428184ff351410136b69b4029dfb96debba2ab2